tests/cucumber/e2e/pageObjects/AddingMoviePage.js

```diff
@@ -0,0 +1,16 @@
+class AddingMoviePage{
+    constructor(){
+        this.pageurlSelector = '//a[normalize-space(text()) = "%s"]'
+        this.moviebuttonSelector = '//div[@id="addTitle"]'
+        this.statusOptionSelector = "#exampleFormControlSelect11"
+        this.userSelector = "//a[@class='nav-link dropdown-toggle text-primary']"
+        this.optionSelector = "//a[@class='dropdown-item' and contains(text(), 'Movie List')]"
+        this.movieStatusTab = "//button[contains(text(), '%s')]"
+        this.movieStatusLocator = "//div[@class='tab-pane fade show active']//a[contains(text(),'%s')] "
+    }
+
+    async moviePage(movie) {
+        await page.locator(formet(this.pageurlSelector,movie)).click()
```

Comment:
```suggestion
        await page.locator(format(this.pageurlSelector,movie)).click()
```
import `format` with `const { format } = require("util")` and you can use format method.